hedera-node/hedera-file-service-impl/src/main/java/module-info.java

```diff
@@ -13,6 +13,7 @@
     requires com.swirlds.config;
     requires org.apache.commons.lang3;
     requires org.apache.logging.log4j;
+    requires org.apache.commons.codec;
```

Comment:
I thought @hendrikebbers had removed this dependency at one point. Are we sure we really want to reintroduce a dependency on apache commons libraries?

Reply:
I need it for DecoderException that is used by mono service during the migration from and to state. this solution is temporary and will be removed afterward.

Reply:
No idea how big the change is and if we can do it in this PR or should create an issue that @povolev15 than can care of to remove it.

Reply:
Mono can be changed to not throw that exception (it's not really the best exception for where it's being thrown regardless).  It's not a small change due to how many places use it, however, so it would need to be a separate issue.

Reply:
new issue created - https://github.com/hashgraph/hedera-services/issues/7133